shared/toks-components/src/components/BackButton/index.tsx

```diff
@@ -1,9 +1,10 @@
+import { Image } from '../Image';
 import { Button } from './style';
 
 export function BackButton() {
   return (
     <Button>
-      <img src="https://asset.tokstudy.com/ic_left_chevron.svg" alt="뒤로가기 버튼 입니다." />
+      <Image src="https://asset.tokstudy.com/ic_left_chevron.svg" alt="뒤로가기 버튼 입니다." />
```

Comment:
```suggestion
      <Icon src="https://asset.tokstudy.com/ic_left_chevron.svg" alt="뒤로가기 버튼 입니다." />
```

이 부분은 Icon assets을 추가해서 Icon svg로 바꿔야 할 것 같다는 의견입니다!

Reply:
이건 추후에 반영해보시죠~

Reply:
옹 지금 확인했습니다 인지하고 있을게요!!